configure

```diff
@@ -3273,6 +3281,15 @@ if test "$disable_tcmalloc" != "yes"; then
   fi
 fi
 print_config "TCMalloc support" "$tcmalloc"
+if ! num "$seed_buckets"; then
+  seed_buckets=4
+elif test "$seed_buckets" -lt 2; then
+  seed_buckets=2
+elif test "$seed_buckets" -gt 16; then
+  seed_buckets=16
+fi
+echo "#define SEED_BUCKETS $seed_buckets" >> $config_host_h
```

Comment:
Looks fine to me, but I think we should call this one CONFIG_SEED_BUCKETS to match with the other entries in configure.

Reply:
I've changed to the CONFIG_SEED_BUCKETS. But, line length is little bit longer in `lib/rand.c`, change also code format in __fill_random_buf.